kotsadm/web/src/components/apps/Dashboard.jsx

```diff
@@ -631,6 +713,39 @@ class Dashboard extends Component {
             </div>
           </Modal>
         }
+
```

Comment:
Wondering if we can move this to a `UpdateCheckerModal {...props} />` component since it's being copied multiple places. Could probably even move most if not all of the methods associated with it to the component to clean up repeated code. Also make it easier to update and maintain.